doc/source/_templates/layout.html

```diff
@@ -0,0 +1,59 @@
+{% extends "basic/layout.html" %}
+
+{% block htmltitle %}
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    {{ super() }}
+{% endblock %}
+
+{% set css_files = [
+    '_static/mergify.css'
+] + css_files %}
+
+{%- block css %}
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  {{ super() }}
+{% endblock %}
+
+{%- block scripts %}
+  {{ super() }}
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+  <script type="text/javascript" src="{{ pathto('_static/mergify.js', 1) }}"></script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css" integrity="sha256-+N4/V/SbAFiW1MPBCXnfnP9QSN3+Keu+NlB+0ev/YKQ=" crossorigin="anonymous" />
+{%- endblock %}
+
+{%- block relbar1 %}{% endblock %}
+{%- block relbar2 %}{% endblock %}
+
+{%- block header %}
+{% include "navbar.html" %}
+{%- endblock %}
+
+{%- block content %}
+  {% include "breadcrumb.html" %}
+  <div class="main container-fluid">
+    <div class="row">
+      <div class="sphinxsidebar col-md-3" aria-label="main navigation">
```

Comment:
I would choice md-2/md-10 instead of md-3/md-9 to have less whitespace between menu and content.